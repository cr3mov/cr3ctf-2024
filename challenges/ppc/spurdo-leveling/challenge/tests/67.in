128097156263197891 970408552724520498
40175 61358
39362 6657
