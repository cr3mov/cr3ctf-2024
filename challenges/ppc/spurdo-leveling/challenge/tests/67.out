258838913575403680 8759382919741 6912162490080
